build_runner/lib/src/package_graph/apply_builders.dart

```diff
@@ -34,33 +38,28 @@ PackageFilter toAll(Iterable<PackageFilter> filters) =>
 PackageFilter toRoot() => (p) => p.isRoot;
 
 /// Apply [builder] to the root package.
-BuilderApplication applyToRoot(Builder builder,
-        {List<String> inputs, List<String> excludes}) =>
-    new BuilderApplication._('', '', [(_) => builder], toRoot(),
-        inputs: inputs, excludes: excludes);
+BuilderApplication applyToRoot(Builder builder) =>
+    new BuilderApplication._('', '', [(_) => builder], toRoot());
```

Comment:
I know this hasn't changed but I am a bit curious as to why we don't need unique builder keys for these builder applications?

Reply:
I'm assuming that they don't need additional configuration from build.yaml so there is nothing to line up the key with. We can revisit this later if we need to.